autoextract/aio/client.py

```diff
@@ -175,10 +202,17 @@ async def request_raw(query: Query,
     ``agg_stats`` argument allows to keep track of various stats; pass an
     ``AggStats`` instance, and it'll be updated.
 
+    Additional ``headers`` for the requests can be provided.
```

Comment:
I think it makes sense to clarify that these are headers sent to the API server, not to remote websites.

Reply:
Make sense. Done here: https://github.com/scrapinghub/scrapinghub-autoextract/pull/24/commits/94484d498961c016d201966366b4076b13f009c9

Reply:
Done here: https://github.com/scrapinghub/scrapinghub-autoextract/pull/24/commits/94484d498961c016d201966366b4076b13f009c9